src/vendor.ts

```diff
@@ -12,6 +12,3 @@ import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/pluck';
 import 'rxjs/add/operator/switchMap';
 import 'rxjs/add/operator/take';
-
-// AngularFire2
-import 'angularfire2';
```

Comment:
Since it's imported in modules in which it is used, including `angularfire2` in vendor.ts is not necessary.